examples/caching_and_replay/README.md

```diff
@@ -26,7 +26,7 @@ To run the model interactively, run ``mesa runserver`` in this directory. e.g.
 
 Then open your browser to [http://127.0.0.1:8521/](http://127.0.0.1:8521/) and press Reset, then Run.
 
-After running a regular simulation, you can **replay** your latest simulation run by first enabling the Replay switch and then pressing Reset.
+After running a regular simulation, you can **replay** your latest cached simulation run by first enabling the Replay switch and then pressing Reset.
```

Comment:
Still no explicit statement that replay needs to be disabled for the first run. You should rephrase to describe the first run on an imperative statement.

Something like:
"First, run the simulation with the Replay switch disabled, ...."

Reply:
This is the only remaining necessary change I requested. Everything else looks good to me.

Reply:
Okay, fixed now.